docs/source/plotting.rst

```diff
@@ -0,0 +1,37 @@
+Visualizing streamz
```

Comment:
Can you please reformat the lines to make them <80chr long?

Reply:
Done.